OpenUtau.Core/Enunu/EnunuConfig.cs

```diff
@@ -12,10 +13,61 @@ class EnunuConfig {
 
         public static EnunuConfig Load(USinger singer) {
             var configPath = Path.Join(singer.Location, "enuconfig.yaml");
-            var configTxt = File.ReadAllText(configPath);
-            RawEnunuConfig config = Yaml.DefaultDeserializer.Deserialize<RawEnunuConfig>(configTxt);
+            var config = new RawEnunuConfig();
+            if (File.Exists(configPath)) {
+                var configTxt = File.ReadAllText(configPath);
+                config = Yaml.DefaultDeserializer.Deserialize<RawEnunuConfig>(configTxt);
+            } else {
+                config = SetSimpleENUNUConfig(singer.Location);
+            }
             return config.Convert();
         }
+
+        public static RawEnunuConfig SetSimpleENUNUConfig(string location) {
+            const string tableExtension = ".table";
+            const string hedExtension = ".hed";
+            const string configYaml = "config.yaml";
+            const string modelPath = "model";
+            var config = new RawEnunuConfig();
+
+            if (Directory.Exists(Path.Combine(location, modelPath))) {
```

Comment:
Code in `if` and `else` are basically the same. You could instead create an array `[location, location/model]`, then loop through it. Again, the directory existence check is unnecessary when you already check the existence of a file in it.